theme/static/css/hydroshare_core.css

```diff
@@ -979,93 +1116,136 @@ span.remove{
     top: 0;
 }
 
-.form-control{
-    margin-bottom:8px;
+.form-control {
+    margin-bottom: 8px;
 }
 
-.no-padding{
-    padding:0;
+.no-padding {
+    padding: 0;
 }
 
-.icon-remove{
+.icon-remove {
     font-size: 17px;
     top: 3px;
     left: 6px;
     color: rgb(217, 83, 79);
-    cursor:pointer;
+    cursor: pointer;
 }
 
-.icon-add{
+.icon-add {
     font-size: 17px;
     top: 3px;
     left: 1px;
     color: #5cb85c;
-    cursor:pointer;
+    cursor: pointer;
 }
 
-.icon-add:hover{
+.icon-add:hover {
     color: #47a447;
 }
 
-.icon-remove:hover{
+.icon-remove:hover {
     color: rgb(168, 69, 66);
 }
 
-#panel-left{
-    padding-right:8px;
-    padding-left:0;
+#panel-left {
+    padding-right: 8px;
+    padding-left: 0;
     margin-bottom: 10px;
 }
 
-#panel-right{
-    padding-left:8px;
-    padding-right:0;
+#panel-right {
+    padding-left: 8px;
+    padding-right: 0;
     margin-bottom: 10px;
 }
-#add-keyword-wrapper{
+
+#add-keyword-wrapper {
     width: 200px;
     display: inline-flex;
 }
 
-
 /* ===== Resource UI changes ===== */
-html, body{
-    height:100%;
+html, body {
+    height: 100%;
     /*background:#F4F4F4;*/
 }
 
-a[data-toggle='tab']{
+a[data-toggle='tab'] {
     /*display: inline-block !important;*/
     overflow: hidden !important;
     text-overflow: ellipsis;
     white-space: nowrap;
 }
 
-#btn-public.active{
-    color:#FFF;
+#btn-public.active {
+    color: #FFF;
     background: #5cb85c;
     border-color: #20833F;
 }
 
-#btn-private.active{
-    color:#FFF;
-    background:#d9534f;
+#btn-private.active {
+    color: #FFF;
+    background: #d9534f;
```

Comment:
Did you intend to make all these unrelated stylistic changes or is that an affect of your editor?